packages/patterns-react/src/patterns/ListSection/ListSection.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { LISTSECTION } from '../../internal/FeatureFlags';
+import ListSectionGroup from './ListSectionGroup';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Section pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.title List section title
+ * @param {string} props.copy List section  short copy to support the title
+ * @param {object} props.listGroup variation of the List section standard, standard with jump link and standard with card link
+ * @returns {object} JSX Object
+ */
+const ListSection = ({ title, copy, listGroup }) =>
+  featureFlag(
+    LISTSECTION,
+    <section
+      data-autoid={`${stablePrefix}--listsection`}
+      className={classNames(`${prefix}--listsection`)}>
+      <div className={`${prefix}--listsection__container`}>
+        <div className={`${prefix}--listsection__row`}>
+          <div className={`${prefix}--listsection__col`}>
+            <h3 className={`${prefix}--listsection__title`}>{title}</h3>
+            <div className={`${prefix}--listsection__content`}>{copy}</div>
+          </div>
+          <div className={`${prefix}--listsection__listgroup__col`}>
+            {_renderListGroup(listGroup)}
+          </div>
+          <div className={`${prefix}--listsection__divider__col`}>
```

Comment:
`${prefix}--listsection__col`, `${prefix}--listsection__listgroup__col`, and `${prefix}--listsection__listgroup__col` all seem to have the same styling, can we just swap all of these to use one classname?

Reply:
also, in mobile breakpoint, the `listsection-group` padding seems off

<img width="535" alt="Screen Shot 2019-10-28 at 9 33 13 AM" src="https://user-images.githubusercontent.com/54281166/67683000-e70ed480-f966-11e9-92b6-896b9d10487c.png">

Reply:
Adjusted padding for mobile view by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/03bb82a14c017854c6ca975d655b1fd8a1465214)